.gitignore

```diff
@@ -29,4 +29,5 @@ storage
 test/e2e/**/packages
 
 # we use corepack, no need to commit yarn binary
-.yarn
+.yarn/*
```

Comment:
with this change, we would start committing the `website/.yarn` folder too, we dont want that

Reply:
I'll fix the globs, good catch